docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     image: mysql:5
     volumes:
       - db_data:/var/lib/mysql
-    restart: always
+    restart: unless-stopped
```

Comment:
This appears to not start the container if it [has been stopped manually](https://docs.docker.com/compose/compose-file/#restart). Are we sure this is not going to cause unexpected results during test runs, for example?

Reply:
I'll remove it, just wanted an alternative to `always`.